### PR TITLE
Document D_PPP_ prefix for macros

### LINKED ISSUE #80: Document D_PP prefix, anything else
Pali knows this undocumented trick, and agreed to update the docs to include it.

Perhaps there are other things as well

### PR DESCRIPTION
**Pull request by pali**
Closes #80

### CODE REVIEW

File: HACKERS [hunk 1]
```diff
@@ -244,6 +244,13 @@ the line C<__UNDEFINED__> to the C<=provides> section to get all macros
 implemented this way to be imported into this section, so they all get
 documented as being provided.
 
+If you need to define helper macro which is not part of C<Devel::PPPort> API
```
Comment (atoomic) [inline HACKERS]: grammar: `define a helper`

File: HACKERS [hunk 2]
```diff
@@ -244,6 +244,13 @@ the line C<__UNDEFINED__> to the C<=provides> section to get all macros
...
+and its usage is only for definition of other C<Devel::PPPort> macros then
+use C<D_PPP_> prefix for this macro name (e.g. C<D_PPP_SVPV_NOLEN_LP_ARG>).
```
Comment (atoomic) [inline HACKERS]: `use the C<D_PPP_> prefix`

File: HACKERS [hunk 3]
```diff
@@ -244,6 +244,13 @@ the line C<__UNDEFINED__> to the C<=provides> section to get all macros
...
+All defined macros are processed by C<Devel::PPPort> and C<Devel::PPPort>
```
Comment (atoomic) [inline HACKERS]: `C<Devel::PPPort> and C<Devel::PPPort>` ?? typo? it's mentioned twice?

Comment (pali) [reply HACKERS]: I rewritten it.

**Comment by atoomic** [comment]
@khwilliamson could you please review these changes?

**Pull request merged**